content/applications/inventory_and_mrp/quality/quality_reporting/quality_alerts_report.rst

```diff
@@ -0,0 +1,194 @@
+linked by a line.
+
+.. image:: quality_alerts_report/line-chart.png
+   :align: center
+   :alt: The line chart graph view on the quality alerts report dashboard.
+
+Pie chart
+~~~~~~~~~
+
+To display quality alert data in a pie chart, click the :icon:`fa-pie-chart` :guilabel:`(Pie Chart)`
+button in the top-left corner of the dashboard.
+
+Each slice of the pie chart represents the quality alerts associated with a specific root cause, in
+a specific stage of the alert review process (ex. workcenter failure alerts in the *In Review*
+stage). The size of each slice represents the number of alerts assigned to it, as a percentage of
+the total number of alerts.
+
+Each slice of the pie appears in a unique color, and a key at the top of the chart denotes the root
+cause and review stage represented by each color.
+
+To remove a slice from the pie chart, click its name in the key at the top of the chart. After doing
+so, the remaining slices expand to fill the space vacated by the removed slice. Click the slice name
+again to make the slice reappear on the chart.
+
+.. image:: quality_alerts_report/pie-chart.png
+   :align: center
+   :alt: The pie chart graph view on the quality alerts report dashboard.
+
+Pivot view
+----------
+
+To display quality alert data in pivot view, click the :icon:`oi-view-pivot` :guilabel:`(Pivot)`
+button in the top-right corner of the dashboard. The pivot view displays data in the form of a
+spreadsheet.
+
+By default, each column is labeled with the name of a quality alert stage (ex. In Review). Since
+alert stages are customizable, they may differ from one database to another.
+
+By default, each row is labeled with the name of a quality alert team.
+
+The cell at the intersection of a column and row displays a number, which signifies the number of
+quality alerts in the column's stage which are assigned to the row's quality alert team.
+
+To change the data displayed by the columns or rows, click the :guilabel:`Total` heading above
+either. Then, click it again to open a popover menu, from which a new variable can be selected.
+
+.. example::
+   Click the :guilabel:`Total` heading above the columns to remove the stage names. Then, click the
+   heading again, and select :guilabel:`Root Cause` from the popover menu. Each column is now titled
+   with the name of a root cause.
+
+.. figure:: quality_alerts_report/pivot-view.png
+   :align: center
+   :alt: The default pivot view on the quality alerts report dashboard.
+
+   The default pivot view on the quality alerts report dashboard.
+
+Kanban view
+-----------
+
+To display quality alert data in a Kanban view, click the :icon:`oi-view-kanban`
+:guilabel:`(Kanban)` button in the top-right corner of the dashboard.
+
+Kanban view displays all existing quality alerts in a Kanban board, with each alert organized into a
+specific stage (ex. In Review). Since kanban stages are customizable, they may differ from one
```

Comment:
```suggestion
specific stage (ex. *In Review*). Since kanban stages are customizable, they may differ from one
```

Reply:
I'm not sure if that kanban gets a capital K - I don't HTINK so since you're describing it in general, and not on our acutal UI. Not 100% sure though.